octobot_trading/modes/abstract_trading_mode.py

```diff
@@ -262,3 +286,68 @@ def get_required_candles_count(cls, tentacles_setup_config: tm_configuration.Ten
             constants.CONFIG_CANDLES_HISTORY_SIZE_KEY,
             common_constants.DEFAULT_IGNORED_VALUE
         )
+
+    @contextlib.asynccontextmanager
+    async def remote_signal_publisher(self, symbol: str):
+        if self.should_emit_trading_signal():
+            try:
+                async with signals.SignalPublisher.instance().remote_signal_bundle_builder(
+                    symbol,
+                    self.get_trading_signal_identifier(),
+                    self.TRADING_SIGNAL_TIMEOUT,
+                    signals.TradingSignalBundleBuilder
+                ) as signal_builder:
+                    yield signal_builder
+            except authentication.AuthenticationRequired as e:
+                self.logger.exception(e, True, f"Failed to send trading signals: {e}")
+        else:
+            yield None
+
+    async def create_order(self, order, loaded: bool = False, params: dict = None, pre_init_callback=None):
+        order_pf_percent = "0"
```

Comment:
Should it be `f"0{script_keywords.QuantityType.PERCENT.value}"` ?

Reply:
yes !